docs/Getting Started.md

```diff
@@ -0,0 +1,778 @@
+# Getting Started
```

Comment:
I think we should avoid spaces, in filenames see above.

Reply:
will change all of them in a separate PR